qa/integration-tests/src/test/java/io/camunda/it/utils/CamundaMultiDBExtension.java

```diff
@@ -158,9 +158,14 @@ public void beforeAll(final ExtensionContext context) {
         databaseSetupReadinessWaitStrategy =
             () -> validateSchemaCreation(DEFAULT_ES_URL, testPrefix, expectedDescriptors);
       }
-      case OS ->
-          multiDbConfigurator.configureOpenSearchSupport(
-              DEFAULT_OS_URL, testPrefix, DEFAULT_OS_ADMIN_USER, DEFAULT_OS_ADMIN_PW);
+      case OS -> {
```

Comment:
:question:  is this enough? when trying to add support for OS in BackupRestoreIT I had to add also some missing configs for OS to this [`MultiDBConfigurator`](https://github.com/camunda/camunda/pull/27029/files#diff-96e42eced6012b5be122d4751010fee42d664c293bf351fe0b402e8344a1ac66)